src/main/java/org/apache/commons/collections4/bloomfilter/WrappedBloomFilter.java

```diff
@@ -25,7 +25,7 @@
  * @since 4.5
  */
 public abstract class WrappedBloomFilter implements BloomFilter {
-    final BloomFilter wrapped;
+    protected final BloomFilter wrapped;
```

Comment:
I personally do not like protected instance variables, it just handcuffs us IMO. I would prefer access through a method and keep the ivar private or package-private.